### ISSUE TITLE
Zoom source list empty on MacOS 13.2 (22D49) Chip Apple M1 Pro

### ISSUE THREAD

**Issue by mimansajaiswal** [tryptech/obs-zoom-and-follow#95]
Hi,

Using the script on my Mac with `macos screen capture` option, it seems like the source is empty. I looked at the code and seems like the type for me is screen_capture. 

So, I changed lines to
`if source_type in { "monitor_capture", "window_capture", "game_capture", "display_capture", "screen_capture" }:`
`if source_type in {'monitor_capture', 'display_capture', 'xshm_input', 'screen_capture'}:`
`mon_show = (True if zoom.source_type in { 'monitor_capture', 'display_capture', 'xshm_input', "screen_capture" } else False)`

Now, it shows up in zoom source but says tuple index should not be string. So, I take a look at `def monitor_capture_mac(self, data)` and print out `monitor` in `monitor in self.monitors.items()`.

And that is is `('Built-in Retina Display', {'id': 1, 'is_primary': True, 'pos': Point(x=0, y=0), 'size': Size(width=1512, height=982), 'workarea': Rect(left=0, top=0, right=1512, bottom=944), 'scale': (200, 200), 'dpi': (144, 144), 'orientation': 0, 'frequency': 120.0, 'colordepth': 32})`

So, I change `monitor` to be `monitor[1]`.

Now the `id` is `1`, so I change `monitor_index` to `1`.

So, now the script does not throw any errors but the zoom is permanently cropped to the top left corner of the screen. Any suggestions?

**Comment by mimansajaiswal** [comment]
Alright, I got it to work.

Step 1: Install pywinctl version 0.0.38. The latter version does not work because it is meant for 3.11 python and OBS does not support 3.11 python.
Step 2: Remain forever confused why OBS **_AND_** pywinctl decide that the macbook screen resolution is 1512*944. That is the source of another half of the problems you are gonna face.
Step 3: Go to obs preferences -> video -> canvas size and set it to your "actual" macbook screen resolution (found in system settings -> general -> about)
Step 4: Add macOS Screen Capture as input to obs.
Step 5: Get to edit transform, set size to "actual" macbook screen resolution, bounding type to "Stretch to bounds" and bounding size to "actual" macbook screen resolution.

Great. Now for the changes I ended up needing to make in the script.

Change these lines to include "screen_capture":
`if source_type in { "monitor_capture", "window_capture", "game_capture", "display_capture", "screen_capture" }:`
`if source_type in {'monitor_capture', 'display_capture', 'xshm_input', 'screen_capture'}:`
`mon_show = (True if zoom.source_type in { 'monitor_capture', 'display_capture', 'xshm_input', "screen_capture" } else False)`

Now we go to `monitor_capture_mac(self, data):`
1. Set default value to 1 in `monitor_index = data.get('display', 1)` instead of 0.
2. Add `monitor = monitor[1]` in `for monitor in self.monitors.items():`

So, what else needed changing? Well, you see pywinctl gets the incorrect resolution too, so we change `get_position = lambda: [c().x, c().y]` to `get_position = lambda: [2*(c().x), 2*(c().y)]` because pywinctl gets exact half of the resolution in my case you see.

We set script properties finally, set zoom source to macOS screen capture and the manual monitor size to "actual" macbook screen resolution.

I think that would be ideally division of $manual_monitor_size/pywinctl_size$

Well, at least it works now!
